slides/belief-that-framework-must-solve-all-the-things.md

```diff
@@ -1,3 +1,13 @@
 ##  Belief That Framework Must Solve All The Things
 
 It's all just JavaScript + software engineering.
+
+Note:
+Henning -> Kahlil
+
+You need to understand that a framework, any framework, will not solve all your
+specific problems. And that is fine.
+As long as it makes you more productive and solves the repetitive, mundane and
+sometimes hard problems for you it is doing its job.
+Chances are that the ecosystem will provide a solution for the more common problems
+that are not covered by the framework itself.
```

Comment:
If you know the the software engineering principles the framework is based on you will know how to deal with the edge cases.